src/hooks/useSAMLSignIn.ts

```diff
@@ -0,0 +1,31 @@
+import { useCallback } from "react";
+import Bugsnag from "@bugsnag/js";
+import firebase from "firebase/app";
+
+export const useSAMLSignIn = (samlAuthProviderId?: string) => {
+  const hasSamlAuthProviderId = samlAuthProviderId !== undefined;
```

Comment:
If the id is empty string this will be true, is this the desired flow? Should be do something like this:
```suggestion
const hasSamlAuthProviderId = !!samlAuthProviderId
```

Reply:
I guess it's the standart way we check if something is not defined

![image](https://user-images.githubusercontent.com/63194656/122254252-9e0dea00-ced5-11eb-95af-00e92111e866.png)

Reply:
Fair enough

Reply:
If it’s an empty string it will just get an error during attempting to signin which will be handled in standard ways